testing/cpputest/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Shiva Velmurugan <shiv@shiv.me>
+# Maintainer: Shiva Velmurugan <shiv@shiv.me>
+pkgname=cpputest
+pkgver=3.8
+pkgrel=1
+pkgdesc="CppUTest is a unit testing and mocking framework for C/C++"
+url="http://cpputest.github.io/"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake"
+install=""
+subpackages="$pkgname-doc"
+source="https://github.com/cpputest/${pkgname}/releases/download/v${pkgver}/${pkgname}-${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver-build/"
+
+build() {
+    mkdir -p "$builddir" && \
+    cd "$builddir" &&\
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr $srcdir/$pkgname-$pkgver
```

Comment:
Variables with file paths which are set outside of the abuild should be always quoted (`"$srcdir"/$pkgname-$pkgver`).